client/landing/stepper/declarative-flow/internals/steps-repository/pattern-assembler/pattern-list-panel.tsx

```diff
@@ -0,0 +1,58 @@
+import AnimateMotion from './animate-motion';
+import useSectionPatternsMapByCategory from './hooks/use-patterns-map-by-category';
+import PatternSelector from './pattern-selector';
+import type { Pattern, Category } from './types';
+import './pattern-list-panel.scss';
+
+type PatternListPanelProps = {
+	onSelect: ( selectedPattern: Pattern | null ) => void;
+	patterns: Pattern[];
+	selectedPattern: Pattern | null;
+	categories: Category[];
+	categorySelected: string | null;
+	openPatternList: boolean | null;
+	onDoneClick?: () => void;
+};
+
+const PatternListPanel = ( {
+	onSelect,
+	patterns,
+	selectedPattern,
+	categories,
+	categorySelected,
+	openPatternList,
+}: PatternListPanelProps ) => {
+	const sectionPatternsMapByCategory = useSectionPatternsMapByCategory( patterns, categories );
+	const categoryPatterns = categorySelected ? sectionPatternsMapByCategory[ categorySelected ] : [];
+
+	return (
+		<AnimateMotion featureName="domAnimation">
```

Comment:
In this way, the framer-motion will be bundled with the PA step and that's why we need to use `AsyncLoad`. It would be better to apply the `AsycLoad` here to keep the size of the PA step smaller. WDYT?

![image](https://user-images.githubusercontent.com/13596067/224246669-a540790e-3611-4a48-a3d2-3eb918db9279.png)

Reply:
Good catch. Because the `PanelListPanel` is only in `CategoryScreenList`, I'll use `AsycLoad` to load `CategoryScreenList` when the flag is enabled. I think that works too, right?

```
<NavigatorScreen path="/homepage/patterns">
  { isEnabled( 'pattern-assembler/categories' ) ? (
    <AsyncLoad
      require="./screen-category-list"
```